universum/modules/vcs/vcs.py

```diff
@@ -20,35 +22,43 @@
 ]
 
 
-def create_vcs(class_type=None):
+def create_vcs(class_type: str = None) -> Type[ProjectDirectory]:
```

Comment:
Considering that the function is used to create base classes, we need to decide what errors we can catch with this annotation.

Reply:
I'd say we can't do much with it right now due to https://github.com/python/mypy/issues/2477

Reply:
I propose to first decide what is the goal of the type checking for this particular case and then maybe refactor vcs.py. For example, we can remove the dynamic base class by switching from the inheritance to composition.